src/main/java/org/folio/oaipmh/client/SourceStorageSourceRecordsClient.java

```diff
@@ -47,10 +41,14 @@ public SourceStorageSourceRecordsClient(String okapiUrl, String tenantId, String
   }
 
   public SourceStorageSourceRecordsClient(String okapiUrl, String tenantId, String token) {
-    this(okapiUrl, tenantId, token, true, 2000, 5000);
+    this(okapiUrl, tenantId, token, true, 2000, 20);
```

Comment:
Instead of hardcoding these timeout values, can we make them constants?

Reply:
Yes, done.